### ISSUE TITLE
Document python.disableInstallationCheck

### ISSUE THREAD

**Issue by brettcannon** [Microsoft/vscode-docs#1589]
https://github.com/Microsoft/vscode-python/blob/5d6493bbea7bbc90c547f14d7a3cb008c372f709/package.json#L1155

Basically turns on/off checks related to your Python install (e.g. whether you're using the system-install of Python on macOS).

**Comment by kraigb** [comment]
Good timing on filing this one...I was just about to commit a number of other changes to the set of April updates. :) Do review the most recent commit for this issue and the other two added settings.

**Comment by brettcannon** [comment]
So the description isn't quite right. All the flag does is disables sanity checks on your Python installation. At the moment that just means warning the user that they are using the Python interpreter that comes with macOS which is known to be bad to develop with.

**Comment by kraigb** [comment]
OK, so setting disableInstallationCheck to true means you don't get a warning if using the OS-installed Python. Is that correct?

**Comment by brettcannon** [comment]
Yep, no warning on macOS about using the system installation. It also turns off the check that there is any Python installation on any platform.

**Comment by brettcannon** [comment]
FYI I just tweaked the wording in the `settings.json` based on the Kraig test for clarity 😉

**Comment by kraigb** [comment]
OK, review the latest commit for edits.

**Comment by brettcannon** [comment]
The "Default behavior" explanation is reversed logic-wise. Since the flag disables the use of `--user` and it's set to `false` by default, that means `--user` is the default behaviour.

**Comment by brettcannon** [comment]
I'm a little worried about the "(as when running on Mac OS)" phrasing giving off the idea that the check is for all system installs no matter what OS and you're mentioning macOS as an example. I know that's technically not what it says, but we have a lot of ESL users where that sophisticated phrasing might throw them.

**Comment by kraigb** [comment]
Sure, that phrase isn't really necessary, then. Check https://github.com/Microsoft/vscode-docs/pull/1579/commits/80345e879cc2d062d9455f547c29fc211741000f.

**Comment by brettcannon** [comment]
Crap, sorry I wasn't clearer, but what I meant was that the check is **only** for the macOS system installation, so I didn't want it to sound like the check applied to any other OS (i.e. Linux gets it right, macOS doesn't 😉 ).

**Comment by kraigb** [comment]
That's why we keep working it. :) So do all aspects of the check apply only to macOS only? It seemed to me that the flag affected all systems in some way, and not just macOS, but perhaps I misinterpreted. In other words, is it _only_ for the check of OS-installed Python on Mac and nothing else, that is, on other systems it doesn't do anything at all?

**Comment by brettcannon** [comment]
No, you're right it affects all system in one way, and macOS in another, but they are separate checks. To put it in outline form (assuming the check is on):
1. Is Python installed _at all_ on **any** OS? (If it isn't we pop up a banner pointing at python.org to install Python)
2. Is the only Python installed for someone running **on macOS** the one that came with macOS? (If it is the only version we pop up a banner basically saying "this isn't a good idea; we would install another copy of Python")

**Comment by kraigb** [comment]
OK, try https://github.com/Microsoft/vscode-docs/commit/4243f327e7a3b455a63aa7f8759c147cdf53e9c8#diff-3d6949cb2b9bcc34f2b8b2a10eba2b17 and https://github.com/Microsoft/vscode-docs/commit/4ad2d1407a2920e22afbf000a623cd069875fe1c#diff-832271fa975d18aff2a7a382b3ba3041.

**Comment by brettcannon** [comment]
Only issue is in the last link there is "disables a warning from the extension if no Python interpreter is." I think "installed" is missing at the end of that sentence.

Otherwise LGTM!
